README.md

```diff
@@ -48,18 +48,18 @@ package manager.
 
 This module supports all editions of Chocolatey, including FOSS, [Professional](https://chocolatey.org/compare) and [Chocolatey for Business](https://chocolatey.org/compare).
 
-This module will ultimately be able to
+This module is able to:
 
-* install Chocolatey
-* work with custom location installations
-* configure Chocolatey
-* use Chocolatey as a package provider
+* Install Chocolatey
+* Work with custom location installations
+* Configure Chocolatey
+* Use Chocolatey as a package provider
 
 ### Why Chocolatey
 
-Chocolatey is a nicer abstraction because it nearly mimics how package managers
-on other operating systems work. If you can imagine the built in provider for
-Windows versus Chocolatey, let's take a look at the use case of installing git:
+Chocolatey is a nicer abstraction because it closely mimics how package managers
```

Comment:
I'm not sure what Chocolately is a "nicer abstration" than. I guess the built-in provider? Anyway, I think you can just lose this phrasing and say "Chocolately closely mimics..."